.orchestra/config/components/mass-testing.yml

```diff
@@ -0,0 +1,98 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/fn-args.lib.yml", "mandatory")
+
+#@yaml/text-templated-strings
+---
+#@ def mass_testing_config(
+#@      report_name=mandatory,
+#@      download_specs=[],
+#@      pre_configure=None,
+#@      post_install=None,
+#@  ):
+builds:
+  default:
+    build_dependencies:
+    - ninja
+    - revng-qa
+    - revng-c
+    - sqlite
+
+    configure: |
+      rm -rf "$BUILD_DIR/build"
+      mkdir -p "$BUILD_DIR/"{inputs,build}
+      cd "$BUILD_DIR"
+
+      BASE_DIR="$ORCHESTRA_DOTDIR/support/mass-testing"
+      export S3_ENDPOINT="$MASS_TESTING_S3_ENDPOINT"
```

Comment:
This is critical but there's no documentation on what this is

Reply:
Added docs to `README.md`